tests/unit/amo/components/TestUserProfileEdit.js

```diff
@@ -0,0 +1,601 @@
+import { oneLine } from 'common-tags';
+import * as React from 'react';
+
+import NotFound from 'amo/components/ErrorPage/NotFound';
+import Link from 'amo/components/Link';
+import UserProfileEdit, {
+  UserProfileEditBase,
+} from 'amo/components/UserProfileEdit';
+import {
+  editUserAccount,
+  fetchUserAccount,
+  finishEditUserAccount,
+  getCurrentUser,
+  loadUserAccount,
+} from 'amo/reducers/users';
+import { USERS_EDIT } from 'core/constants';
+import { ErrorHandler } from 'core/errorHandler';
+import ErrorList from 'ui/components/ErrorList';
+import Notice from 'ui/components/Notice';
+import {
+  dispatchClientMetadata,
+  dispatchSignInActions,
+} from 'tests/unit/amo/helpers';
+import {
+  createFakeEvent,
+  createStubErrorHandler,
+  createUserAccountResponse,
+  fakeI18n,
+  shallowUntilTarget,
+} from 'tests/unit/helpers';
+
+
+describe(__filename, () => {
+  const defaultUserProps = {
+    biography: 'Saved the world, too many times.',
+    display_name: 'Matt MacTofu',
+    homepage: 'https://example.org',
+    location: 'Earth',
+    occupation: 'Superman',
+    userId: 500,
+    username: 'tofumatt',
+  };
+
+  function createFakeEventChange({ name, value }) {
+    return createFakeEvent({
+      currentTarget: {
+        name,
+        value,
+      },
+    });
+  }
+
+  function signInUserWithUsername(username) {
+    return dispatchSignInActions({
+      userProps: { ...defaultUserProps, username },
+    });
+  }
+
+  function renderUserProfileEdit({
+    i18n = fakeI18n(),
+    params = { username: 'tofumatt' },
+    userProps = { ...defaultUserProps },
+    store = null,
```

Comment:
Why do you need this? Without it won't the test below for `!store` still work, as `store` will be `undefined`? If you could remove this you could also remove the eslint comment below.

Reply:
Do you mean getting rid of the `= null` or the whole line?

Reply:
I meant getting rid of the whole line, but maybe that won't work?

Reply:
If the line gets removed, then we will have to check/assign `props.store`.